resources/views/components/user-card.blade.php

```diff
@@ -0,0 +1,16 @@
+<div class="p-6 flex space-x-2 shadow-sm rounded-lg" style="background-color:white;margin:1rem;width:400px;">
+    <div class="flex-1">
+        <div class="flex justify-between py-6">
+            <div>
+                <a href="/user/{{ $user->name }}">
+                    <img width='100' height="100"src="{{$user->profile_image_url}}">
+                </a>
+            </div>
+            <div class="ml-4">
+                <a href="/user/{{ $user->name }}">{{ $user->name }}</a>
+                <p>User Created: {{ $birthday }}</p>
```

Comment:
change user created to "First Chirped", and in the component constructor the birthday should get the created date of the first chirp the user sent. if the user has not sent any chirps birthday should be null

change total chirps to $chirpcount .  " Chirps"

if the user hasn't chirped this should say "Just Hatched"

use @if directive for this with a bool component variable $hasChirps which is set in the component class

Reply:
Just Hatched!